chichiwang/exercise-js
# Move `pages/Layout` to `app/Layout`

## Description
Linked to Issue: #113 
Layout is an application-level concern, not a page-level concern, so the `Layout` component is being moved from `pages/Layout` to `app/Layout`.

## Changes
* Directory `pages/Layout/` is moved to `app/Layout/`
* References to `Layout` are updated in `app/dataRoutes/index.ts` and `app/dataRoutes/__tests__/index.test.js`

## Steps to QA
* Pull down and switch to this branch
* Run the application and verify in browser that that the pages still look correct, with the layout applied
